systers_portal/systers_portal/settings/base.py

```diff
@@ -127,6 +127,7 @@
 # https://django-allauth.readthedocs.org/en/latest/#configuration
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_ADAPTER = 'users.adapter.SystersUserAccountAdapter'
+ACCOUNT_EMAIL_VERIFICATION = "mandatory"
```

Comment:
This is magic 🌟 but has limitation. What this is doing is picking the text before ```@example.com``` and make it as a username. In our case we allow the user to have a username different than this. In the messages we need to have the username which the user provided.

Reply:
Oops. I didn't realize the username is changing. Thanks for pointing out. I will modify this.